nsv13/code/modules/overmap/armour/nano_well.dm

```diff
@@ -358,7 +358,7 @@ Starting Materials
 		return
 	var/adjust = text2num(params["adjust"])
 	if(action == "power_allocation")
-		if(adjust && isnum(adjust))
+		if(isnum(adjust))
```

Comment:
while at it, may aswell just CLAMP the value in the lines below (and then return) instead of doing some unneccessary if filtering